### PR TITLE
Fix broken link in README.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Fix broken link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ The [compile configuration system](https://docs.mbed.com/docs/mbed-os-handbook/e
 
 ___Inspecting the configuration___
 
-If the program uses the [compile configuration system](https://github.com/ARMmbed/mbed-os/blob/master/docs/config_system.md), you can use `mbed compile --config` to view the configuration:
+If the program uses the [compile configuration system](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/d), you can use `mbed compile --config` to view the configuration:
```
Comment (bridadan) [inline README.md]: Looks an extra `/d` snuck in at the end of the link?

Comment (AnotherButler) [inline README.md]: Good catch

**Comment by AnotherButler** [comment]
No idea.

**Comment by screamerbg** [comment]
Thanks! :)

**Pull request merged**